package io.gdcc.xoai.dataprovider.exceptions;